src/components/AppHeader.vue

```diff
@@ -152,13 +162,46 @@ export default defineComponent({
 							"Successful",
 							`Your DAO App of ID ${this.daoID} is selected.`
 						);
+						if (this.walletStore.address) {
+							isApplicationOpted(this.walletStore.address, this.daoID as number)
```

Comment:
avoid `as number`. daoID can be undefined as well. Put a check for daoID undefined.

Reply:
![image](https://user-images.githubusercontent.com/100185149/167597332-2ae6f0ce-8399-45e1-a6b8-47787d42cd7d.png)
we are already testing it before

Reply:
ok